packages/v4/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -0,0 +1,168 @@
+
+### Event handler parameter 
+
+Previously, our event handler parameters weren’t consistently ordered. To meet industry standards, and get everything on the same page, we’ve updated all callback signatures for component event handling properties so that the event is the first parameter. This change will make it easier to incorporate third party libraries, especially in the case of form building tools, like Formik.
+
+Our [Codemods](/get-started/upgrade#run-our-codemods) should take care of most of these changes, but if your callback function is defined at the class level you will need to manually update your code.
+
+### Icons
+
+To improve the consistency of icon usage across projects adopting PatternFly, we are no longer using inline styles, color, or size for icons. Now, React icons come with a built-in class `pf-v5-svg`, which gives an icon a default height, width, and color.
+
+If you want a custom color or size, wrap your icon in the [`<Icon>` React component](/components/icon), which comes with colors, size, etc.
+
+If you are not importing `@patternfly/react-styles`, but are still using `@patternfly/react-icons` you will need to provide the generic styles that you have separated from the icons and moved into `@patternfly/react-styles`. For more information, [refer to the @patternfly/react-icons README file.](https://github.com/patternfly/patternfly-react/blob/main/packages/react-icons/README.md)
+
+## Component updates and enhancements
+
+### Card
+
+The following updates were made to the card component to improve the experience for both developers and end-users.
+
+**Card header**
+
+Our previous card API was difficult to understand and utilize. We reconfigured its markup and React subcomponents so you can implement cards more easily.
+
+As a result, `CardHeaderMain` and `CardActions` are no longer exported from PatternFly, and are instead rendered internally within the `CardHeader` sub-component. Any content and properties for `CardHeaderMain` or `CardActions` should be passed directly to [`CardHeader`](https://www.patternfly.org/v4/components/card#cardheader) instead.
+
+**Clickable and selectable cards**
+
+Clickable and selectable cards have been redesigned to improve usability and accessibility.
+
+With our previous implementation of cards, screen readers were unable to determine when a card was clickable or selectable. Similarly, users navigating without a screen reader would still have to mouse over a card to determine when it was clickable or selectable. We weren’t satisfied with these hurdles.
+
+To improve accessibility and usability for both keyboard and screen reader users, our new [selectable and clickable cards](/components/card#selectable) are redesigned to help users determine when a card can be selected.
+
+Previous design:
```

Comment:
I think that for this purpose I would use images that are in the design guidelines vs the React examples. I like the image here for example: https://patternfly-org-pr-3610-site.surge.sh/components/card/design-guidelines#cards-in-catalog-views  It does better at placing this in context.

Reply:
Could you help clarify what you're thinking? Do you mean using just this one image of the cards in catalog views instead of a before or after? Or grabbing a before/after from different places?

Reply:
I'm not sure. If you want to show before and after you can use this image of what it looks like now in PF4: https://www.patternfly.org/v4/components/card/design-guidelines#cards-in-catalog-views

Reply:
Not sure if it helps but I mocked up one of our selectable card demos from PF4 to look like PF5 

PF5 selectable card
<img width="1739" alt="Screenshot 2023-06-21 at 11 07 56 AM" src="https://github.com/patternfly/patternfly-org/assets/35148959/d4d8fc10-1434-4503-b8bc-771b14078205">

PF4 selectable card
<img width="1739" alt="Screenshot 2023-06-21 at 11 08 26 AM" src="https://github.com/patternfly/patternfly-org/assets/35148959/ec2a7aa6-34a0-412f-b03e-2814043d82b2">

Reply:
This is very helpful, thank you!